### PR TITLE
Various italian fixes

### PR DESCRIPTION
**Pull request by Annalisanephila**
Update italian translation of the django girls tutorial

### CODE REVIEW

File: it/code_editor/instructions.md [hunk 1]
```diff
@@ -12,11 +12,11 @@ Gedit è un editor open-source e gratuito, disponibile per tutti i sistemi opera
 
 Sublime Text è uno tra gli editor più utilizzati. Ha un periodo di prova gratuito. È molto facile da installare e da utilizzare ed è disponibile per tutti i sistemi operativi.
 
-[Scaricalo qui](https://www.sublimetext.com/2)
+[Scaricalo qui](http://www.sublimetext.com/2)
```
Comment (aniav) [inline it/code_editor/instructions.md]: Hi, why did you change the link to be http only here and in other places? It's always better to use https :)

File: it/css/README.md [hunk 2]
```diff
@@ -10,18 +10,18 @@ Ma non vogliamo ricominciare da capo, giusto? Ancora una volta useremo qualcosa
...
 
 ## Installa Bootstrap
 
 Per installare Bootstrap, avrai bisogno di aggiungere questo nel tag `<head>` del tuo file `.html` (`blog/templates/blog/post_list.html`):
 
-```html
-<link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap.min.css">
-<link rel="stylesheet" href="//maxcdn.bootstrapcdn.com/bootstrap/3.2.0/css/bootstrap-theme.min.css">
-```
+    html
```
Comment (aniav) [inline it/css/README.md]: The fencing should definitelly stay. It's a more desired syntax of code highlighting.

**Comment by aniav** [comment]
@Annalisanephila I believe you had some merge leftovers so I am not looking through the whole pull request. Will you have time to cleanup the changes so we will be able to go through them? Thanks! 👍

**Comment by patjouk** [comment]
Thanks for this PR :sparkles: 
@DjangoGirls/it-pr can you review it? Thanks!

**Comment by Annalisanephila** [comment]
I will but I don't have the write access

**Comment by yakky** [comment]
@Annalisanephila I'll help you to get this in shape for merging

**Comment by olasitarska** [comment]
As this hasn't been reviewed for quite a while now, I'm going to close it. If anyone still wants to work on this, please reopen :)
